File: JAVASCRIPT/1180.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let N = parseFloat(lines.shift());
let l2 = lines.shift();
    let Y = l2.split(" ");
    let menor = 998;
    let posicao = 0;
    
    for(let i = 0; i < N; i++) {
        let x = parseFloat(Y.shift());
        if(x < menor) {
            
            menor = x;
            posicao = i;
     }
             
     }
     console.log("Menor valor: " + menor);
     console.log("Posicao: " + posicao);

File: JAVA/1180.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int N = sc.nextInt();
		int menor = sc.nextInt();
		int posicao = 0;
		for(int i = 1; i < N; i++) {
		int x = sc.nextInt();
		if(x < menor){
		menor = x;
		posicao = i;

		}
		}
		System.out.println("Menor valor: "+ menor);
		System.out.println("Posicao: "+ posicao);
		}

		}
		